RecoParticleFlow/PFClusterProducer/interface/PFEcalEndcapRecHitCreator.h

```diff
@@ -0,0 +1,142 @@
+#ifndef RecoParticleFlow_PFClusterProducer_PFEcalEndcapRecHitCreator_h
```

Comment:
After the last simplification, the PFEcalEndcapRecHitCreator.h now overlaps to a rather significant extent with the implementation of PFEcalBarrelRecHitCreator.h
So, the two can be merged with a template specialization as we had before